cmd/ocm/delete/ingress/cmd.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"fmt"
+	"regexp"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	c "github.com/openshift-online/ocm-cli/pkg/cluster"
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+)
+
+var ingressKeyRE = regexp.MustCompile(`^[a-z0-9]{3,5}$`)
```

Comment:
Since you're not tracking the `api` "ingress", this regexp can be narrowed down to `{4,5}` (btw, thanks for skipping the `api` "ingress", I'm not super thrilled at having mixed it with the ingress and I think it belongs in an `edit cluster` command instead). :+1: